ops/ansible/roles/bfd-pipeline/templates/bfd-pipeline-service.sh.j2

```diff
@@ -34,6 +34,9 @@ export RDA_GRPC_PORT='{{ data_pipeline_rda_grpc_port }}'
 export RDA_GRPC_MAX_IDLE_SECONDS='{{ data_pipeline_rda_grpc_max_idle_seconds }}'
 export RDA_GRPC_AUTH_TOKEN='{{ data_pipeline_rda_grpc_auth_token }}'
 
+# Referenced by the pipeline's logback.xml file to add environment name for use in splunk.
+export BFD_ENV_NAME='{{ env_name_std|default("unknown-environment", true) }}'
```

Comment:
I think the preference is to add the defaults here https://github.com/CMSgov/beneficiary-fhir-data/blob/master/ops/ansible/roles/bfd-pipeline/defaults/main.yml

Reply:
I was hesitant to make any change to the `env_name_std` variable itself (even setting a default) since it's not one that I created.  I can certainly put at value in there though.